### PR TITLE
Issue 2: fedev | Personal Page

### PR DESCRIPTION
**Pull request by fedeloterstein**
## Description

In this PR we worked on:
- Creating the builders folder
- Creating my personal page
- Support for dark mode.

 
Captura de pantalla 2024-09-10 a la(s) 21 07 09
Captura de pantalla 2024-09-10 a la(s) 21 07 21

## Additional Information

- [x] I have read the [contributing docs](/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) (if this is your first contribution)
- [x] This is not a duplicate of any [existing pull request](https://github.com/scaffold-eth/scaffold-eth-2/pulls)

## Related Issues

_Closes #9 

_Note: If your changes are small and straightforward, you may skip the creation of an issue beforehand and remove this section. However, for medium-to-large changes, it is recommended to have an open issue for discussion and approval prior to submitting a pull request._

Your ENS/address: 0xDFbE6c0A54F9f4f758753aE56eDD02Dd92C29be3

### CODE REVIEW

**Review by phipsae — CHANGES_REQUESTED** [review]
@fedeloterstein, great job on the clean and well-organized code for your personal page! 

Just a small suggestion—consider adding a personal touch, like a picture (it doesn’t necessarily have to be of you), to make it stand out even more.

I've also left a minor comment below.

File: packages/nextjs/contracts/deployedContracts.ts [hunk 1]
```diff
@@ -4,6 +4,296 @@
  */
 import { GenericContractsDeclaration } from "~~/utils/scaffold-eth/contract";
 
-const deployedContracts = {} as const;
+const deployedContracts = {
```
[the remark this answers is not in the document]
Comment (fedeloterstein) [reply packages/nextjs/contracts/deployedContracts.ts]: I already corrected it.

**Comment by fedeloterstein** [comment]
> @fedeloterstein, great job on the clean and well-organized code for your personal page!
> 
> Just a small suggestion—consider adding a personal touch, like a picture (it doesn’t necessarily have to be of you), to make it stand out even more.
> 
> I've also left a minor comment below.

I've already made an update. I liked this way of keeping a clean profile, not so overloaded. Sometimes less is more. 
Captura de pantalla 2024-09-13 a la(s) 00 44 55

**Comment by phipsae** [comment]
> > @fedeloterstein, great job on the clean and well-organized code for your personal page!
> > Just a small suggestion—consider adding a personal touch, like a picture (it doesn’t necessarily have to be of you), to make it stand out even more.
> > I've also left a minor comment below.
> 
> I've already made an update. I liked this way of keeping a clean profile, not so overloaded. Sometimes less is more. Captura de pantalla 2024-09-13 a la(s) 00 44 55

Thanks! I agree on that :) 

Merged.

**Pull request merged**